main/src/ca/uwaterloo/flix/language/dbg/prettierPrettyPrinting/DocAstFormatter.scala

```diff
@@ -0,0 +1,356 @@
+package ca.uwaterloo.flix.language.dbg.prettierPrettyPrinting
+
+import ca.uwaterloo.flix.language.dbg.prettierPrettyPrinting.Doc._
+import ca.uwaterloo.flix.language.dbg.prettierPrettyPrinting.DocAst.Type
+
+import scala.annotation.tailrec
+
+object DocAstFormatter {
+
+  def format(p: DocAst.Program)(implicit i: Indent): List[Doc] = {
+    import scala.math.Ordering.Implicits.seqOrdering
+
+    implicit val i: Indent = indentationLevel(4)
+    val DocAst.Program(enums0, defs0) = p
+    val enums = enums0.map {
+      case DocAst.Enum(_, _, sym, cases) =>
+        val delimitedCases = semiSepOpt(cases.map {
+          case DocAst.Case(sym) =>
+            text("case") +: text(sym.toString) :: text("(?)")
```

Comment:
Why (?) here?

Reply:
Enums are a mess right now in the backend, id have to extract the fields from `tpeDeprecated` so I thought we could leave this WIP until that is fixed instead of introducing dependencies